README.md

```diff
@@ -21,6 +41,7 @@ Install all packages.  Uses Yarn workspaces.
 ```sh
 yarn install
 ```
+#### Running the tests
```

Comment:
Change this tag to `#### Launching a local server for a package`, and move the #### Running the tests down to right above the test command.